larva/src/main/java/org/frankframework/larva/queues/QueueWrapper.java

```diff
@@ -82,76 +81,22 @@ public static QueueWrapper create(IConfigurable configurable) {
 		return new QueueWrapper(configurable);
 	}
 
-	public synchronized PipeLineSession getSession() {
-		if(!containsKey(PIPELINESESSION_KEY)) {
-			PipeLineSession session = new PipeLineSession();
-			put(PIPELINESESSION_KEY, session);
-			return session;
-		}
-		return (PipeLineSession) get(PIPELINESESSION_KEY);
-	}
-
-	public ListenerMessageHandler getMessageHandler() {
-		return (ListenerMessageHandler) get(MESSAGE_HANDLER_KEY);
-	}
-
-	public void setSenderThread(SenderThread senderThread) {
-		put(SENDER_THREAD_KEY, senderThread);
-	}
-
-	public SenderThread getSenderThread() {
-		return (SenderThread) get(SENDER_THREAD_KEY);
-	}
-
-	public void removeSenderThread() {
-		remove(SENDER_THREAD_KEY);
-	}
-
-	public QueueWrapper invokeSetters(Properties queueProperties) {
-		return invokeSetters(-1, queueProperties);
-	}
-
-	public QueueWrapper invokeSetters(int defaultTimeout, Properties properties) {
-		if(containsKey(MESSAGE_HANDLER_KEY)) {
-			if(defaultTimeout > 0) {
-				getMessageHandler().setTimeout(defaultTimeout);
-			}
-
-			QueueUtils.invokeSetters(getMessageHandler(), properties); //set timeout properties
-		}
-
-		String convertException = properties.getProperty(CONVERT_MESSAGE_TO_EXCEPTION_KEY);
-		put(CONVERT_MESSAGE_TO_EXCEPTION_KEY, Boolean.valueOf(convertException));
-
-		mapParameters(properties);
-
-		return this;
-	}
-
-	private void mapParameters(Properties properties) {
-		if(get() instanceof IWithParameters) {
-			Map<String, Object> paramPropertiesMap = createParametersMapFromParamProperties(properties, true, getSession());
-			Iterator<String> parameterNameIterator = paramPropertiesMap.keySet().iterator();
-			while (parameterNameIterator.hasNext()) {
-				String parameterName = parameterNameIterator.next();
-				Parameter parameter = (Parameter)paramPropertiesMap.get(parameterName);
-				((IWithParameters) get()).addParameter(parameter);
-			}
-		}
-	}
-
 	public static Map<String, Object> createParametersMapFromParamProperties(Properties properties, boolean createParameterObjects, PipeLineSession session) {
```

Comment:
Reworked this method a bit, same as in `LarvaTool`. The rest of the file is formatting